.github/workflows/lint.yaml

```diff
@@ -0,0 +1,24 @@
+name: lint
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
Use standard workflow.

eg https://github.com/lsst/daf_butler/blob/main/.github/workflows/lint.yaml

(but feel free to not use ruff -- that's up to you)

Reply:
Updated